ref/vk/vk_render.h

```diff
@@ -76,6 +76,9 @@ typedef enum {
 	// - studio: kRenderTransAdd
 	kVkRenderType_1_1_R,    // blend: src + dst, depth test
 
+	// Decals changing diffuse and rmxx of surface material
+	kVkRenderType_Decal,    // blend: src*a + dst (1-a), no depth test or write
```

Comment:
Инконгруэнтный комментарий. В коде `ci.depthTestEnable = VK_TRUE;`
```suggestion
	kVkRenderType_Decal,    // blend: src*a + dst (1-a), depth test, no depth write
```